src/MCT.js

```diff
@@ -79,7 +79,6 @@ import Browse from './ui/router/Browse.js';
 export class MCT extends EventEmitter {
   constructor() {
     super();
-    EventEmitter.call(this);
```

Comment:
Whaaaat were we even doing here? Suspect this was a hangover from prototypical inheritance that was left in by mistake.

Reply:
Yup, exactly.